projects/core/src/lib/colors/color-palette-editor/color-palette-editor.component.scss

```diff
@@ -0,0 +1,49 @@
+.tabs-viewport {
+    display: flex;
+    height: 20rem;
+    flex-flow: row wrap;
+}
+
+geoengine-color-attribute-input {
+    flex-grow: 1;
+}
+
+.removeButton {
+    margin-left: 10px;
+    -ms-transform: translateY(10%);
+    transform: translateY(10%);
```

Comment:
What is this?
And why is there a `-ms` special prefix?

This looks wrong.

Reply:
But is it correctly aligned?

With translate:

![grafik](https://user-images.githubusercontent.com/4947061/234017410-0d87ec41-c4ff-454b-97f8-37650470df7c.png)

Without translate:

![grafik](https://user-images.githubusercontent.com/4947061/234017592-fe449d8f-41a8-43be-ac25-cf76709b8c3a.png)

Moreover, please re-check your set height of 3rem… it just isn't true:

![grafik](https://user-images.githubusercontent.com/4947061/234019543-9a553457-383c-4c80-8731-65aac550e248.png)

Reply:
In [d98a53d](https://github.com/geo-engine/geoengine-ui/pull/614/commits/d98a53d6ab45317fbf6372c25b93c6285a420327) I adjusted the row template to `grid-template-rows: 3rem auto;` and removed the misleading height property on `grid-colorpicker`, which only has its natural height now.
Setting `align-items: center` unfortunately doesn't seem to align the color picker properly, so I kept the `margin-top` to align the remove button.